Makefile

```diff
@@ -98,7 +98,14 @@ inkscape:
 # register icon for Pd mimetype (adapted from https://stackoverflow.com/a/31836/1964109 )
 mime-register:
 	xdg-icon-resource install --context mimetypes --size 512 pd-512.png x-puredata
-	xdg-mime install x-puredata.xml 
+	xdg-mime install x-puredata.xml
+	cp pd-gui.desktop ~/.local/share/applications/pd-gui.desktop
```

Comment:
this doesn't look right.
- do not hardcode the location. instead use a variable that allows to install the desktop-file into the user-home or system-wide
- do not hard-code `~/.local/share/`; instead use `${XDG_DATA_HOME}` (if set).

these two comments might look the same, but they are really orthogonal